sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsPartitionManagement.java

```diff
@@ -79,7 +80,9 @@ void createPartition(
      * @return true if the partition exists, false otherwise
      */
     default boolean partitionExists(InternalRow ident) {
```

Comment:
This PR also changed this API. In the past, we coud use this API to check wether table had partitions

Reply:
After the changes, you can do the same, right?